crates/storage/src/connection/block.rs

```diff
@@ -13,20 +13,23 @@ pub(super) fn insert_block_header(
 
     // Insert the header
     tx.inner().execute(
-        r"INSERT INTO starknet_blocks 
-                   ( number,  hash,  root,  timestamp,  gas_price,  sequencer_address,  version_id,  transaction_commitment,  event_commitment,  class_commitment)
-            VALUES (:number, :hash, :root, :timestamp, :gas_price, :sequencer_address, :version_id, :transaction_commitment, :event_commitment, :class_commitment)",
+        r"INSERT INTO headers 
```

Comment:
I don't like `headers` too much -- I would have expected something like `block_headers` which is less ambiguous (for me).